db/patterns/ethyca.eno

```diff
@@ -0,0 +1,14 @@
+name: Ethyca
+category: consent
+website_url: https://ethyca.com/
+organization: ethyca
+
+--- domains
+ethyca.com
+--- domains
+
+--- filters
+||ethyca.com^$3p
```

Comment:
That is the same as `||ethyca.com^$third-party` (only the short form)

Reply:
This looks good thanks- we use a handful of subdomains like `fides.ethyca.com`, `api.ethyca.com`, and `cdn.ethyca.com` to serve this up so I think this should cover everything. I prefer being comprehensive like that so we're transparent

Reply:
Makes sense. We currently have no clear rule when to use top-level domain vs listing sub domains. As a rule of thumb: if there is a huge number sub domains, it is best to merge. Also, sometimes sub domains are created for individual customers and - even though DNS entries are public - we try not to list them.

I think, here it best to start like that; but we can change to listing subdomains later if the current solution makes problems, or if you prefer it the other way.

Reply:
Thanks @philipp-classen. Since I'm "self-submitting" here specifically to ensure our scripts/APIs don't get miscategorized as advertising/etc, I'm OK going more broad! If we see a need to recategorize something (e.g. `cdn.ethyca.com` as a CDN!) we can probably do that, but overall "Consent" feels like a good default for what we do since we're also out there helping ensure data privacy 😄